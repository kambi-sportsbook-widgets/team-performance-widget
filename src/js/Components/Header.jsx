import React from 'react';

const Header = ({ title }) => {
   return (
      <header className="kw-header">{title}</header>
   );
};

Header.propTypes = {
   /**
    * Header's title
    */
   title: React.PropTypes.string
};

Header.HEIGHT = 50;

export default Header;
