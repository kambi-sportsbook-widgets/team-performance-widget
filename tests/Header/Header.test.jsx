/* eslint-env jest */
import React from 'react';
import ReactTestUtils from 'react-addons-test-utils';
import { shallow } from 'enzyme';
import Header from '../../src/js/Components/Header';

let renderer;

describe('Header DOM rendering', () => {
   beforeEach(() => {
      renderer = ReactTestUtils.createRenderer();
   });
   it('renders correctly', () => {
      expect(renderer.render(
         <Header title='Title' />
      )).toMatchSnapshot();
   });
});
