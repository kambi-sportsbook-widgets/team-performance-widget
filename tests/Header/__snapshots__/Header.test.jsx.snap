// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header DOM rendering renders correctly 1`] = `
<header
  className="kw-header KambiWidget-card-support-text-color"
>
  Title
</header>
`;
